pkg/R/pairRDD.R

```diff
@@ -0,0 +1,799 @@
+# Pair RDD in R implemented in S4 OO system.
```

Comment:
This comment can be changed to something like `Actions and transformations supported on RDDs contains pairs (i.e. key, value)`